### DISCUSSION TITLE
What editor should we use?

### DISCUSSION THREAD

**Discussion by MikeDev101** [NovaoTech/solaris#5] [Development]
As mentioned in https://github.com/JTechnologies/Polaris/projects/1#card-64583208, we should use E羊icques.

However, wouldn't Turbowarp be a better option? They both share the custom extension / direct file loading functionality and Turbowarp has better performance optimization features.

**Comment by icarusinthewings** [comment]
At the time that I wrote that, I had no idea that Turbowarp had an editor. Since we are now using Turbowarp for project loading, we will also be using Turbowarp's editor if possible. However, we will have some fun with licensing because Turbowarp is LGPL3 and this project is MIT. I might consider switching to the LGPL, but the only downside is that LGPL may not be compatible with some app stores.

**Reply by icarusinthewings** [reply]
I updated the card to use turbowarp instead. I've already contacted GarboMuffin on scratch and he has given me some pointers.
